doc/classes/SceneTree.xml

```diff
@@ -34,7 +35,8 @@
 			<argument index="2" name="method" type="StringName">
 			</argument>
 			<description>
-				Calls [code]method[/code] on each member of the given group, respecting the given [enum GroupCallFlags].
+				Calls [code]method[/code] on each member of the given group, respecting the given [enum GroupCallFlags]. You can pass arguments to [code]method[/code] by specifying them at the end of the method call.
+				[b]Note:[/b] By default, [method call_group_flags] will call methods with an one-frame delay, in a way similar to [method Object.call_deferred]. To call methods immediately, use [method call_group_flags] with the [constant GROUP_CALL_REALTIME] flag.
```

Comment:
IIUC for this one it's not "by default" as the flags parameter is not optional, so you always have to make a choice.

Reply:
Instead, you could clarify in the previous sentence that the flags are used to control when the call happens and in what order.